### ISSUE TITLE
Change Weather to Night in 0.8.4

### ISSUE THREAD

**Issue by mhusseinsh** [carla-simulator/carla#862]
Hello
I am working with CARLA release 0.8.4

Is there a way to have a night weather ? Is there a place where I can add or change weather settings ?

**Comment by Romanenko-Serhii** [comment]
Yes, look this https://github.com/carla-simulator/carla/issues/525

**Comment by mhusseinsh** [comment]
@TheNihilisticRobot @nsubiron any help here please ?

**Comment by Romanenko-Serhii** [comment]
@mhusseinsh He, sorry for the late response: https://carla.readthedocs.io/en/0.8.4/carla_settings/

**Comment by mhusseinsh** [comment]
@Romanenko-Serhii No problem, thank you

Sorry I don't get your point, what does this has to do with my issue ? I want to achieve a night mode, but I am directly using the 0.8.4 release, which is already built (so I have the executable file directly), this means, that I don't have Config files and I cannot do the solution which is done here #525

**Comment by Romanenko-Serhii** [comment]
@mhusseinsh Yes, sorry I forget to add, that you need use full version of the 0.8.4 release, not prebuilt.

**Comment by mhusseinsh** [comment]
@Romanenko-Serhii so there is no way around at all to use the prebuilt ?

**Comment by Romanenko-Serhii** [comment]
@mhusseinsh I think no because you need to make configuration of the server.

**Comment by analog-cbarber** [comment]
Not sure how meaningful night mode would be without dynamic headlights, street lights and lighting in the buildings.

**Comment by mhusseinsh** [comment]
@analog-cbarber I am talking about night mode as sun wise and sky color wise
No problem with street lights and lighting in buildings or cars, I want a normal night mode like what it is in real life, dark sky and no sun

like these examples
![image](https://user-images.githubusercontent.com/33177438/47069938-7039ea00-d1f0-11e8-9a31-cb5ba2f66d6b.png)
![image](https://user-images.githubusercontent.com/33177438/47069951-7cbe4280-d1f0-11e8-8039-2218b921dfa1.png)
![image](https://user-images.githubusercontent.com/33177438/47069961-82b42380-d1f0-11e8-85ad-ce74bb3f2561.png)

**Comment by analog-cbarber** [comment]
Those examples clearly have dynamic lights and have building lighting, so I don't understand what you are asking for. If you turn off all sky lighting you will be driving around in the dark without lights. That's my point. Unless you also add support for dynamic headlights, street lights and modify the building meshes to show window lighting, you will not get anything like what you see above, and none of those will happen magically by playing with the weather settings.

**Comment by mhusseinsh** [comment]
@analog-cbarber so in CARLA, there is no option to turn off sky lighting, and turn on street lights or car lights ? this never happens ?

**Comment by analog-cbarber** [comment]
I don't believe there are any lights to turn on. You would have to create dynamic lights for every vehicle, and I don't believe they have done that yet, but you should go ahead and play with the weather settings and see if it gives you something acceptable.

**Comment by mhusseinsh** [comment]
@analog-cbarber yes I will try anyways, thanks

@Romanenko-Serhii when I add a new weather in the weather config file, so this for example will be weatherID 16, do I need to rebuild again and create a new executable file ? or it will automatically be activated ?
